components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/base/base_container.html

```diff
@@ -195,6 +195,61 @@
         {% if page_size %}
             var PAGE_SIZE = {{ page_size }};
         {% endif %}
+
+
+        // ** "Open With" config used by E.g. ome.tree.js **
+        WEBCLIENT.OPEN_WITH = [];
+        {% if ome.open_with %}
+            // This is a json dump, so we can add it directly without escaping
+            WEBCLIENT.OPEN_WITH = {{ ome.open_with|safe }};
+        {% endif %}
+        // Helper can be used by 'open with' plugins to add isEnabled()
```

Comment:
Could these two functions be added to the JavaScript library rather than being placed here in the template?